apps/fortuna/src/keeper.rs

```diff
@@ -128,9 +128,80 @@ impl Default for KeeperMetrics {
 }
 
 impl KeeperMetrics {
-    pub async fn new(registry: Arc<RwLock<Registry>>) -> Self {
+    pub async fn new(
+        registry: Arc<RwLock<Registry>>,
+        chain_ids: Vec<String>,
+        provider_addresses: Vec<Address>,
+    ) -> Self {
         let mut writable_registry = registry.write().await;
         let keeper_metrics = KeeperMetrics::default();
+        
+        // Initialize metrics for each chain_id and provider_address pair
```

Comment:
move this loop to the end of the method

please also leave a comment giving instructions to someone adding a new metric. they need to register the metric *and* add a get_or_create call in the loop